src/MessagePack.UnityClient/Assets/Scripts/MessagePack/Resolvers/StandardResolver.cs

```diff
@@ -79,6 +79,39 @@ static FormatterCache()
         }
     }
 
+    /// <summary>
+    /// The standard resolver to use when targeting AOT platforms or when avoiding runtime code generation is undesirable for perf reasons.
+    /// </summary>
+    /// <remarks>
+    /// <para>
+    /// This resolver <em>may</em> invoke reflection at runtime, but only for types that are not known at compile time.
+    /// </para>
+    /// <para>
+    /// This resolver is meant to be combined with a <c>GeneratedResolver</c> produced by our source generator.
+    /// This is done automatically in the <c>GeneratedResolver.InstanceWithStandardAotResolver</c> property.
+    /// </para>
+    /// </remarks>
+    public static class StandardAotResolver
+    {
+        /// <summary>
+        /// The singleton instance that can be used.
+        /// </summary>
+        public static readonly IFormatterResolver Instance = CompositeResolver.Create(
+            new IMessagePackFormatter[]
+            {
+                ExpandoObjectFormatter.Instance,
+            },
+            new IFormatterResolver[]
+            {
+                BuiltinResolver.Instance, // Try Builtin
+                AttributeFormatterResolver.Instance, // Try use [MessagePackFormatter]
+#if UNITY_2018_3_OR_NEWER
+                MessagePack.Unity.UnityResolver.Instance,
+#endif
+                ImmutableCollection.ImmutableCollectionResolver.Instance,
```

Comment:
It seems that this diff adds an Immutable reference to Unity.  
https://github.com/AArnott/MessagePack-CSharp/pull/67 or https://github.com/AArnott/MessagePack-CSharp/pull/68 correspondence should be done